malware/get_samples.py

```diff
@@ -15,13 +15,15 @@
 from urllib2 import urlopen
 from urlparse import urlunparse
 
+
 def get_app_creds():
     """Reads your App ID and App Secret from env variables"""
     return {
-        'AppID' : os.environ['FB_THREATEXCHANGE_APP_ID'],
-        'AppSecret' : os.environ['FB_THREATEXCHANGE_APP_SECRET'],
+        'AppID': os.environ['FB_THREATEXCHANGE_APP_ID'],
+        'AppSecret': os.environ['FB_THREATEXCHANGE_APP_SECRET'],
```

Comment:
These should probably be changed to use the vocabulary and pull the proper environment variables, similar to how init does it.